lib/db.ts

```diff
@@ -0,0 +1,9 @@
+import mongoose from 'mongoose';
+
+const url = process.env.MONGODB_URI as string;
+let connection: typeof mongoose;
+
+const startDb = async () => {
+    if (!connection) connection = await mongoose.connect(url)
```

Comment:
Let's implement curly eslint rule

https://eslint.org/docs/latest/rules/curly

> JavaScript allows the omission of curly braces when a block contains only one statement. However, it is considered by many to be best practice to never omit curly braces around blocks, even when they are optional, because it can lead to bugs and reduces code clarity. So the following: